test/unit/utils/random.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+exports.key = length => {
+  const buf = []
+  const chars = 'abcdefghijklmnopqrstuvwxyz0123456789'
+  for (let i = 0; i < length; ++i) {
+    buf.push(chars[exports.randomInt(0, chars.length - 1)])
+  }
+
+  return buf.join('')
+}
+exports.randomUuid = () => {
+  // See:
+  // http://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript
```

Comment:
I guess it is ok to have our implementation rather to rely on 3rd party library as it is used in tests

Reply:
I copy-pasted it from frontend